src/pages/debrief/cat-be/debrief.cat-be.selector.ts

```diff
@@ -131,6 +131,21 @@ export const getVehicleCheckDrivingFaults =
     return result;
   };
 
+export const vehicleChecksExist = (vehicleChecks: CatBEUniqueTypes.VehicleChecks): boolean => {
```

Comment:
Why do we need to check if vehicle checks exist? We can't progress the WRTC page if they don't and this is on debrief.

Reply:
It is possible to end a test without entering anything against vehicle checks. This selector is used to check if we have a response or not and ultimately this will display the vehicle checks component or not.

Reply:
refactored to use lodash as agreed for clarity.